chain/yoda/context.go

```diff
@@ -13,10 +13,19 @@ import (
 	"github.com/bandprotocol/bandchain/chain/yoda/executor"
 )
 
+type FeeEstimationData struct {
+	minCount    int64
+	callData    []byte
+	validators  []string
```

Comment:
We used only size of the validator?

Reply:
yes, that's make sense. At first I think size of validator addresses might be variant.